doc/ledger3.texi

```diff
@@ -4635,13 +4635,16 @@ Ledger will include @samp{; transid: 767718} in the first transaction
 from the file above.
 
 @findex --invert
+@findex --auto-match
 @findex --account @var{STR}
 @findex --rich-data
 
-The @command{convert} command accepts three options. They are
-@option{--invert} which inverts the amount field, @option{--account
-@var{STR}} which you can use to specify the account to balance against,
-and @option{--rich-data} which stores additional metadata as tags.
+The @command{convert} command accepts four options. They are
+@option{--invert} which inverts the amount field, @option{--auto-match}
+which automatically match account from ledger journal for every CSV
+line, @option{--account @var{STR}} which you can use to specify the
+account to balance against, and @option{--rich-data} which stores
+additional metadata as tags.
```

Comment:
I've checked what the output would look like. From my point of view, the /info/ output is ugly, the /pdf/ output is not very nice, and the /html/ output is somewhat nicer with @enumerate. I prefer to stay as-is